### DISCUSSION TITLE
How to create an export that includes the customer number?

### DISCUSSION THREAD

**Discussion by vic-t** [kimai/kimai#2350] [Questions & Answers] [answered]
I would like for the customer number to be shown in the CSV export. If I'm not mistaken, that's not possible with the default export settings so I could use some guidance about the steps needed to change this. 

The documentation explains that there are export templates and export renderers. So, if I understand this correctly, I actually need a different renderer. But it's also possible to just create a new template because 

> Internally for each template a new ExportRenderer service is registered, called exporter_renderer.filename_EXT_twig

Which way is simpler, to create a new template or to create a new renderer? Can you provide me a rough outline of the steps necessary to achieve my goal? 

Thanks!

**Comment by kevinpapst** [comment, marked as answer]
Spreadsheets don't have templates, only HTML and PDF have.

Same topic as #2353 - please include the column in the feature request
